### ISSUE TITLE
Building grocy fails on nixos-unstable

### ISSUE THREAD

**Issue by lokegustafsson** [NixOS/nixpkgs#94798]
**Describe the bug**
Grocy (services.grocy) fails to build on recent nixpkgs versions.

**To Reproduce**
The following (minimal example) container fails to build (on [commit](https://github.com/NixOS/nixpkgs/commit/8e2b14aceb1d40c7e8b84c03a7c78955359872bb), and also current master)
```nix
{
  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";

  outputs = { self, nixpkgs }: {

    nixosConfigurations.container = nixpkgs.lib.nixosSystem {
      system = "x86_64-linux";
      modules = [
        ({ pkgs, ... }: {
          boot.isContainer = true;
          services.grocy = {
            enable = true;
            hostName = "localhost";
            nginx.enableSSL = false;
          };
        })
      ];
    };
  };
}
```
with the output:
```
error: --- Error ---------------------------------------------------------------------------------------------------------------------- nix
builder for '/nix/store/6lwrbia06zbv62zjmd4076gr806d3d2d-php-intl-7.3.20.drv' failed with exit code 2; last 10 log lines:
  locale_lookup.phpt() [tests/locale_lookup_variant2.phpt]
  =====================================================================

  You may have found a problem in PHP.
  This report can be automatically sent to the PHP QA team at
  http://qa.php.net/reports and http://news.php.net/php.qa.reports
  This gives us a better understanding of PHP's behavior.
  If you don't want to send the report immediately you can choose
  option "s" to save it.        You can then email it to qa-reports@lists.php.net later.
  Do you want to send this report now? [Yns]: make: *** [Makefile:134: test] Error 1
error: --- Error ---------------------------------------------------------------------------------------------------------------------- nix
1 dependencies of derivation '/nix/store/23cj880pgwnm4a5c35y77asg6w2c3bz0-php.ini.drv' failed to build
error: --- Error ---------------------------------------------------------------------------------------------------------------------- nix
1 dependencies of derivation '/nix/store/p2bcsz7mh6dw8ynac44n6kzsqrfqsnbl-php-with-extensions-7.3.20.drv' failed to build
error: --- Error ---------------------------------------------------------------------------------------------------------------------- nix
1 dependencies of derivation '/nix/store/knzvf9xlq02y14xj26zyw32d2hglvfdn-unit-phpfpm-grocy.service.drv' failed to build
error: --- Error ---------------------------------------------------------------------------------------------------------------------- nix
1 dependencies of derivation '/nix/store/c5az39b06hwnzss6h098vjhji6lf5px0-system-units.drv' failed to build
error: --- Error ---------------------------------------------------------------------------------------------------------------------- nix
1 dependencies of derivation '/nix/store/26mhwj441c28myj3mnb0cq8gc9i1srjl-etc.drv' failed to build
error: --- Error ---------------------------------------------------------------------------------------------------------------------- nix
1 dependencies of derivation '/nix/store/2as9vpwv2067h8h3bi2y2n8ygkhkysg8-nixos-system-nixos-20.09.20200805.8e2b14a.drv' failed to build
```

This is obviously unexpected. Curiously, the build succeeds on nixos-20.03, but I haven't figured out to override service versions in a flake and can't really use that as I depend on unstable for other things.

**Notify maintainers**
@Ma27 

**Metadata**
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: 
# a list of nixos modules affected by the problem (am I doing this correctly?)
module: nixos/modules/services/web-apps/grocy.nix
```

**Comment by lokegustafsson** [comment]
I found [this](https://hydra.nixos.org/build/124676672), indicating it failed quite recently. Still have no idea why though.

**Comment by Ma27** [comment]
`grocy` itself builds, `phpPackages.intl` is the problem.
cc @NixOS/php

**Comment by Ma27** [comment]
The problem seems to be the ICU bump in 4afdb415d2147d60f1fb2aa24dc07c14fa4f6b59

**Comment by Ma27** [comment]
Fixed in cecc4ce36184fb1051ac6924ebf5d65b5e0668aa.
